README.md

```diff
@@ -31,6 +31,11 @@ The mini-lab is a small, virtual setup to locally run the metal-stack. It deploy
 Here is some code that should help you to set up most of the requirements:
 
  ```bash
+# If UFW enabled.
+# Disable the firewall or allow traffic through Docker network IP range. 
+sudo ufw status
+sudo ufw allow from 172.17.0.0/16 
```

Comment:
Sorry to be picky, but the Prefix to allow should be 172.16.0.0/12 according to https://www.rfc-editor.org/rfc/rfc1918

Reply:
Wow, thanks!